dlp/api/Snippets.Tests/JobsGetTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Google.Cloud.Dlp.V2;
+using System.IO;
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class JobsGetTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture Fixture { get; }
+        public JobsGetTests(DlpTestFixture fixture)
+        {
+            Fixture = fixture;
+        }
+
+        [Fact]
+        public void TestGetJob()
+        {
+            using var randomBucketFixture = new RandomBucketFixture();
```

Comment:
You should be able to just replace this block with 

`var dlpJob = dlp.CreateDlpJob(Fixture.GetTestRiskAnalysisJobRequest());`

It doesn't need to be an inspect job, and risk analysis jobs are simpler to set up. It makes the test a little clearer.

You can see the example here: https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/master/dlp/api/Snippets.Tests/JobsListTests.cs